### PR TITLE
Add template.json for translation, update index.html and SD extension i18n

### PR DESCRIPTION
**Pull request by PasserDreamer**
2. update SD extension dropdown menu i18n.
3. update zh-tw.json.
4. add a template.json for translation.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -109,7 +109,7 @@ <h4 class="margin0"><span data-i18n="kobldpresets">Kobold Presets</span>
                             </div>
                             <div id="novel_api-presets">
                                 <h4 class="margin0">
-                                    <span data-i18n="novelaipreserts">NovelAI Presets</span>
+                                    <span data-i18n="novelaipresets">NovelAI Presets</span>
```
Comment (Cohee1207) [inline public/index.html:112]: You have to fix this in all i18n files

Comment (PasserDreamer) [reply public/index.html:112]: done.

**Comment by Cohee1207** [comment]
> Add template.json for translation

Who will be responsible for keeping it up to date? This either needs to be an automated process, or not present in the repo, because I'm not a big manually adding new strings to it.

**Comment by PasserDreamer** [comment]
No one. But it can be outdated because it won't be used directly. If you think we shouldn't keep even the most basic version, then just delete it.

**Comment by Cohee1207** [comment]
> If you think we shouldn't keep even the most basic version, then just delete it.

I do. Unless there's some sort of automation, it's going to be a PITA to support. See this thread #2098

**Pull request merged**